sources/procfs.go

```diff
@@ -125,68 +125,68 @@ func newLustreProcMetric(filename string, promName string, source string, path s
-			{"stats", "write_minimum_size_bytes", writeMinimumHelp},
-			{"stats", "write_maximum_size_bytes", writeMaximumHelp},
-			{"stats", "write_total_bytes", writeTotalHelp},
-			{"sync_journal", "sync_journal", "Binary indicator as to whether or not the journal is set for asynchronous commits"},
-			{"tot_dirty", "tot_dirty", "Total number of exports that have been marked dirty"},
-			{"tot_granted", "tot_granted", "Total number of exports that have been marked granted"},
-			{"tot_pending", "tot_pending", "Total number of exports that have been marked pending"},
+			{"blocksize", "ost_blocksize_bytes", "Filesystem block size in bytes"},
+			{"brw_size", "ost_brw_size_total", "Block read/write size in bytes"},
+			{"brw_stats", "ost_pages_per_bulk_rw_total", pagesPerBlockRWHelp},
+			{"brw_stats", "ost_discontiguous_pages_total", discontiguousPagesHelp},
+			{"brw_stats", "ost_disk_io_now", diskIOsInFlightHelp},
+			{"brw_stats", "ost_io_time_ms", ioTimeHelp},
+			{"brw_stats", "ost_disk_io_total", diskIOSizeHelp},
+			{"degraded", "ost_degraded_now", "Binary indicator as to whether or not the pool is degraded - 0 for not degraded, 1 for degraded"},
+			{"filesfree", "ost_files_free_now_total", "The number of inodes (objects) available"},
+			{"filestotal", "ost_file_capacity", "The maximum number of inodes (objects) the filesystem can hold"},
+			{"grant_compat_disable", "ost_grant_compat_disabled", "Binary indicator as to whether clients with OBD_CONNECT_GRANT_PARAM setting will be granted space"},
+			{"grant_precreate", "ost_grant_precreate_capacity_bytes", "Maximum space in bytes that clients can preallocate for objects"},
+			{"job_cleanup_interval", "ost_job_cleanup_interval_seconds", "Interval in seconds between cleanup of tuning statistics"},
+			{"job_stats", "ost_job_read_samples_total", readSamplesHelp},
+			{"job_stats", "ost_job_read_minimum_size_bytes", readMinimumHelp},
+			{"job_stats", "ost_job_read_maximum_size_bytes", readMaximumHelp},
+			{"job_stats", "ost_job_read_bytes_total", readTotalHelp},
+			{"job_stats", "ost_job_write_samples_total", writeSamplesHelp},
+			{"job_stats", "ost_job_write_minimum_size_bytes", writeMinimumHelp},
+			{"job_stats", "ost_job_write_maximum_size_bytes", writeMaximumHelp},
+			{"job_stats", "ost_job_write_bytes_total", writeTotalHelp},
+			{"job_stats", "ost_job_getattr_total", getattrHelp},
+			{"job_stats", "ost_job_setattr_total", setattrHelp},
+			{"job_stats", "ost_job_punch_total", punchHelp},
+			{"job_stats", "ost_job_sync_total", syncHelp},
+			{"job_stats", "ost_job_destroy_total", destroyHelp},
+			{"job_stats", "ost_job_create_total", createHelp},
+			{"job_stats", "ost_job_statfs_total", statfsHelp},
+			{"job_stats", "ost_job_get_info_total", getInfoHelp},
+			{"job_stats", "ost_job_set_info_total", setInfoHelp},
+			{"job_stats", "ost_job_quotactl_total", quotactlHelp},
+			{"kbytesavail", "ost_kilobytes_available", "Number of kilobytes readily available in the pool"},
```

Comment:
Do we want to add `_now` to keep this consistent with others (such as line 136)?